### PR TITLE
Added marquee to TwitchPopups

### PR DESCRIPTION
**Pull request by MojoChin**
Text will scroll along popupbox if too large for screen.

Not up to the code warrior standards

IT JUST WORKS!

### CODE REVIEW

File: twitchpopups.htm [hunk 1]
```diff
@@ -100,28 +108,50 @@ <h1 id="popuptext"></h1>
...
                 }
             }
                 
     }
         
     // Animate text
     function doAnimation() {
-        var textWidth = $("#popuptext").width();
+        $('#popuptext').stopMarquee(); //stop marquee if new alert
+	textWidth = $("#popuptext").width();
```
Comment (rnsc) [inline twitchpopups.htm:120]: Indentation is wrong

File: twitchpopups.htm [hunk 2]
```diff
@@ -100,28 +108,50 @@ <h1 id="popuptext"></h1>
...
+        $("#popupbox").animate({width:animateWidth}, 500);
         $("#popuptext").animate({"opacity":1, "margin-left":"15px"}, 700);
+	leftMovement = 0;
     }
     
     function doHotseatAnimation() {
-        var textWidth = $("#popuptext").width();
+	$('#popuptext').stopMarquee(); //stop marquee if new alert
+        textWidth = $("#popuptext").width();
+		animateWidth = Math.min(textWidth+30, 1880); //get smallest width between screen width and text width
```
Comment (rnsc) [inline twitchpopups.htm:140]: Indentation issue

**Comment by rnsc** [comment]
Use spaces instead of tabs for indentation :)

**Comment by DaftLimmy** [comment]
Thanks but I'll be updating with a version that fits all the text without scrolling.

**Pull request closed without merging**